_remake/main.js

```diff
@@ -59,16 +59,30 @@ if (RemakeStore.isMultiTenant()) {
       next();
     } else {
       // handle api endpoints, rendered routes and resources
-      let splitString = req.get("host").split(".");
-      let validSplit = splitString.length === 3;
-      let appName = splitString[0] || "";
-      let validAppName = /^[a-z]+[a-z0-9-]*$/.test(appName);
+      const hostParts = req.get("host").split(".");
```

Comment:
You don't want to use this library here?

https://www.npmjs.com/package/parse-domain

Reply:
Will have a look